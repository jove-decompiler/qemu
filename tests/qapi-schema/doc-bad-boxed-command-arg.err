doc-bad-boxed-command-arg.json:9: the following documented members are not in the declaration: a
