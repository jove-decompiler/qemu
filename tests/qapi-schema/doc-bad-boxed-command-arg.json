# Boxed arguments are not to be documented with the command
# BUG: not rejected

##
# @Args:
# @a: an argument
##
{ 'struct': 'Args', 'data': { 'a': 'int' } }

##
# @cmd-boxed:
# @a: bogus
##
{ 'command': 'cmd-boxed', 'boxed': true,
  'data': 'Args' }
