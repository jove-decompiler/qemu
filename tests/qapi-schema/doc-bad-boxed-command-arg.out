module None
object q_empty
enum QType
    prefix QTYPE
    member none
    member qnull
    member qnum
    member qstring
    member qdict
    member qlist
    member qbool
module doc-bad-boxed-command-arg.json
object Args
    member a: int optional=False
command cmd-boxed Args -> None
    gen=True success_response=True boxed=True oob=False preconfig=False
doc symbol=Args
    body=

    arg=a
an argument
doc symbol=cmd-boxed
    body=

    arg=a
bogus
